tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -75,9 +78,60 @@ spec:
       asgs=$((nodes/max_nodes))
       echo "asgs: $asgs"
       node_group=$(params.nodegroup-prefix)$(params.cluster-name)-nodes
+      
+      create_and_validate_launchTemplate_resource()
+      {
+        launch_template_name=$node_group-launchTemplate
+        iam_instance_profile=$NODE_ROLE_ARN
+        imageId=/aws/service/eks/optimized-ami/$(params.kubernetes-version)/amazon-linux-2/recommended/image_id
+        
+        cat sed "s/\$LAUNCH_TEMPLATE_NAME/$launch_template_name/g; \
+        s/\$NODE_INSTANCE_PROFILE/$iam_instance_profile/g; \
+        s/\$IMAGE_ID/$imageId/g;" > "$(workspaces.config.path)/cfnConfig.yaml" <<EOL
+        AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
can we not embed this and pull it in from assets like we had discussed earlier ? Like how we do for [vpc](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/assets/amazon-eks-vpc.json)

Reply:
This LT has parameters that we are including for each run like Kubernetes version, node arn which needs to be included once the task is running. Nathan is on the same page too. We can discuss further if there are specific values that can be hardcoded? 

Or do you mean we can add this template in the assets and add these values when we curl? Let me know either ways

Reply:
>do you mean we can add this template in the assets and add these values when we curl?
This is what i meant ^^^.
Curl it like how we pull vpc and substitute values that you wanted in that file in the task if it can't be generalized. So, task is much more flexible and readable.

Reply:
Think of it in terms of, if I bring in another custom LT, how can I reuse this task for that so it's not tightly coupled ?

Reply:
Added the LT to assets